.github/workflows/deploy-contract.yml

```
@@ -70,8 +72,15 @@ jobs:
           TOKEN: ${{ inputs.token }}
           DURATION: ${{ inputs.duration }}
           HARDHAT_IGNITION_CONFIRM_DEPLOYMENT: false
-      - name: Get contract address
+      - name: Get sepolia contract address
+        if: ${{ inputs.network == 'sepolia' }}
         run: |
           ADDRESS=$(jq -r '."Deployment#Raffle"' ignition/deployments/chain-11155111/deployed_addresses.json)
           echo '# Raffle deployed! 🚀' >> $GITHUB_STEP_SUMMARY
           echo "Address: <a href=\"https://sepolia.etherscan.io/address/$ADDRESS\">$ADDRESS</a>" >> $GITHUB_STEP_SUMMARY
+      - name: Get mainnet contract address
+        if: ${{ inputs.network }} == 'mainnet'
```

actionlint

if: condition "${{ inputs.network }} == 'mainnet'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)